playbook/workshops/dodge/add_enemy_sprite.md

```diff
@@ -1,5 +1,8 @@
 # Add Enemy Sprite
 
+_Questions, comments and critiques are welcome! Send your feedback
```

Comment:
Well, there are two types of feedback we want to collect it seems:

1) Minor things that are wrong on the current step that they are on
2) Overall feedback on the workshop

**Currently** 

The [current form](https://docs.google.com/forms/d/1IxbiDtyP-UOx3hRGu3o2I-iVll95xQ6I_pW8JS3TZ2k/viewform?entry.1677546962=Adding+an+enemy+sprite+to+the+bullet+dodging+game) seems to address "Overall feedback on the workshop". I say that this looks like the "Overall" feedback based on looking at the questions on the form:

> 1. What are you working on?
> 2. What is one thing that we could improve on?
> 3. What is one thing that we did well?
> 4. Anything else you would like to share?

The above set of questions seems to gather Overall feedback. Gathering "Overall" feedback seems to make more sense at the **end** of a workshop because I don't want to answer all these questions in the middle of a workshop.

**Actionable Task**

It seems to make sense to move this to the **end** of the workshops. For Dodge, I would insert it [like this](https://gist.github.com/jonleung/bc562f12e8be126c3e00):

**Capturing Intermediary Feedback**

However, to capture intermediate feedback on the workshop as they work on the tutorial (like in `add_enemy_sprite.md`), we might want to make it as easy as possible by only have 1 question that's layed out like the following:

1) [Here's how it would be linked in the readme:](https://gist.github.com/jonleung/6fba5d2d47ef41d39923)
2) The question on the form might be "Find anything confusing or frustrating? Let us know here)"